scw_serverless/utils/loader.py

```diff
@@ -0,0 +1,52 @@
+import importlib.util
+import inspect
+import sys
+from pathlib import Path
+
+from scw_serverless.app import Serverless
+
+
+def get_module_name(file: Path) -> str:
+    """Extract the module name from the path."""
+
+    file = file.resolve()
+    return (
+        str(file.relative_to(Path(".").resolve())).removesuffix(".py").replace("/", ".")
+    )
+
+
+def get_app_instance(file: Path) -> Serverless:
+    """Load the app instance from the client module."""
+
+    module_name = get_module_name(file)
+    parent_directory = str(file.parent.resolve())
+
+    spec = importlib.util.spec_from_file_location(
+        module_name,
+        str(file.resolve()),
+        submodule_search_locations=[parent_directory],
+    )
+
+    if not spec or not spec.loader:
+        raise ImportError(
+            f"Can't find module {module_name} at location {file.resolve()}"
+        )
+
+    user_app = importlib.util.module_from_spec(spec)
+    sys.path.append(parent_directory)
```

Comment:
This is pretty much the actual fix. Before executing the module, add the package directory to PYTHONPATH so that imports will be also looked for here.